yb-voyager/cmd/importData.go

```diff
@@ -295,11 +321,12 @@ func applyTableListFilter(importFileTasks []*ImportFileTask) []*ImportFileTask {
 	checkUnknownTableNames(excludeList, "exclude")
 
 	for _, task := range importFileTasks {
-		if len(includeList) > 0 && !slices.Contains(includeList, standardizeCaseInsensitiveTableNames(task.TableName)) {
+		defaultSourceSchema, _ := getDefaultSourceSchemaName() 
+		if len(includeList) > 0 && !slices.Contains(includeList, standardizeCaseInsensitiveTableNames(task.TableName, defaultSourceSchema)) {
```

Comment:
Line too long.

Factor out following duplicated statement:
`tableName := standardizeCaseInsensitiveTableNames(task.TableName, defaultSchemaName)`
Pass `tableName` to the `slice.Contains()` calls.